sklearn/naive_bayes.py

```diff
@@ -669,18 +677,10 @@ class MultinomialNB(BaseDiscreteNB):
     class_log_prior_ : array, shape (n_classes, )
         Smoothed empirical log probability for each class.
 
-    intercept_ : array, shape (n_classes, )
```

Comment:
do not remove them yet. you should use the sphinx directive `.. deprecate:: 0.22` after the explanation of the parameter and warn that the parameter is deprecated and it will be removed (with the right version)